src/main/scala/uk/gov/hmrc/AccessibilityLinterPlugin.scala

```diff
@@ -47,24 +49,35 @@ object AccessibilityLinterPlugin extends AutoPlugin with LibraryManagementSyntax
       a11yExtract.value
       npmProcess("npm install failed for a11y linter")(a11yRoot.value / "js", "install")
     }
-    val a11yTest = taskKey[Unit]("Performs sbt test after installing the required npm assets")
+
+    val accessibilityAssessment = taskKey[Unit]("Runs tests in the accessibility folder after installing the required npm assets\"")
+
+    lazy val A11yTest = config("a11y") extend Test
+
+    lazy val a11yTestSettings: Seq[Setting[_]] =
+      inConfig(A11yTest)(Defaults.testSettings) ++
+        Seq(unmanagedSourceDirectories in A11yTest := (baseDirectory in A11yTest)(base => Seq(base / "a11y")).value)
   }
 
   import autoImport._
 
   // This adds a value for the settingKey
   override lazy val globalSettings: Seq[Setting[_]] = Seq.empty
 
+  // This adds the accessibility test configuration
+  override val projectConfigurations: Seq[Configuration] = Seq(A11yTest)
+
   // This adds implementation for the taskKeys and additional library dependencies
   override lazy val projectSettings: Seq[Setting[_]] = Seq(
     a11yRoot := a11yRootTask.value,
     a11yExtract := a11yExtractTask.value,
     a11yInstall := a11yInstallTask.value,
-    a11yTest := (test in Test).dependsOn(a11yInstall).value,
+    accessibilityAssessment := (test in A11yTest).dependsOn(a11yInstall).value,
+    Test / testOptions += Tests.Setup( () => a11yInstall.value ),
```

Comment:
I think this should be `A11yTest / testOptions` or it will run also with just `sbt test`

Reply:
I have made the change, thank you.